bridge/engine/EngineMixer.cpp

```diff
@@ -4861,10 +4863,10 @@ bool EngineMixer::asyncHandleSctpControl(const size_t endpointIdHash, memory::Un
 
 bool EngineMixer::asyncRemoveRecordingStream(const EngineRecordingStream& engineRecordingStream)
 {
-    const bool postResult =
-        post(utils::bind(&EngineMixer::removeRecordingStream, this, std::cref(engineRecordingStream)));
-    _messageListener.asyncRecordingStreamRemoved(*this, engineRecordingStream);
-    return postResult;
+    return post([this, &engineRecordingStream]() {
+        this->removeRecordingStream(engineRecordingStream);
+        _messageListener.asyncRecordingStreamRemoved(*this, engineRecordingStream);
```

Comment:
Tells to Mixer that can delete the recoding stream, then it need to be called by engine thread after event has been processed